api_spec.yaml

```diff
@@ -144,6 +145,12 @@ paths:
                     properties:
                       status:
```

Comment:
ok status should return number of auth code deleted as well?

Reply:
changes made